tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -464,6 +540,8 @@ private void AddInstrumentationsAndProducts(bool fromRemoteConfig)
             {
                 AsmRemoteConfigurationProducts.AsmDataProduct.ConfigChanged += AsmDataProductConfigChanged;
                 AsmRemoteConfigurationProducts.AsmProduct.ConfigChanged += AsmProductConfigChanged;
+                AsmRemoteConfigurationProducts.AsmDataProduct.ConfigRemoved += AsmDataProductConfigRemoved;
```

Comment:
can `AsmDDProduct` config product be removed too? would that mean falling back to the embedded ruleset? (could only be the local one as otherwise asm_dd wouldnt be advertised as capability if there was a local rule file